application/classes/Ushahidi/Repository/Contact.php

```diff
@@ -60,11 +60,10 @@ protected function setSearchConditions(SearchData $search)
 	{
 		$query = $this->search_query;
 
-		$user = $this->getUser();
-
-		// Limit search to user's records unless they are admin
-		if (! $this->isUserAdmin($user)) {
```

Comment:
I don't think we should be removing this. Though maybe we should update it to check a permission as well as isUserAdmin()